packages/e2e/cypress/e2e/api/sqlRunner.cy.ts

```diff
@@ -92,6 +93,97 @@ Object.entries(warehouseConnections).forEach(
                     expect(fieldResults.order_date).to.be.eq('date');
                 });
             });
+
+            it(`Get streaming results from ${warehouseName}`, () => {
+                const [database, schema] = getDatabaseDetails();
+                const selectFields =
+                    warehouseConfig.type !== WarehouseTypes.SNOWFLAKE
+                        ? `*`
+                        : `payment_id as "payment_id", 
+                amount as "amount", 
+                payment_method as "payment_method"`; // Need to lowercase column ids in snowflake
+                const sql = `SELECT ${selectFields}
+                     FROM ${database}.${schema}.payments ORDER BY payment_id asc LIMIT 2`;
+
+                cy.request({
+                    url: `${apiUrl}/projects/${projectUuid}/sqlRunner/run`,
+                    headers: { 'Content-type': 'application/json' },
+                    method: 'POST',
+                    body: JSON.stringify({
+                        sql,
+                    }),
+                }).then((runResp) => {
+                    expect(runResp.status).to.eq(200);
+                    const { jobId } = runResp.body.results;
+
+                    const maxRetries = 20; // Snowflake is a bit slow compared to the others (normally takes ~3 seconds)
+
+                    // Poll request until job is `completed`
+                    const poll = (retries = 0) => {
+                        cy.wait(500);
+                        cy.request({
+                            url: `${apiUrl}/schedulers/job/${jobId}/status`,
+                            method: 'GET',
+                        }).then((resp) => {
+                            expect(resp.status).to.eq(200);
+                            cy.log(
+                                `Job status (${retries}): ${resp.body.results.status}`,
+                            );
+                            if (resp.body.results.status !== 'started') {
+                                expect(resp.body.results.status).to.eq(
+                                    'completed',
+                                );
+                                const { fileUrl } = resp.body.results.details;
+                                cy.request({
+                                    url: fileUrl,
+                                    method: 'GET',
+                                }).then((fileResp) => {
+                                    expect(fileResp.status).to.eq(200);
+                                    const lines = fileResp.body
+                                        .trim()
+                                        .split('\n');
+                                    const results = lines.map((line) =>
```

Comment:
this is how we can consume a JSONL, not sure if there is a better way (JSON.load?)